R/methods.R

```diff
@@ -2838,6 +2842,9 @@ setGeneric("test_gene_enrichment", function(.data,
 																							 .contrasts = NULL,
 																								method = c("camera" ,    "roast" ,     "safe",       "gage"  ,     "padog" ,     "globaltest",  "ora" ),
 																							 species,
+																							 msigdb.gsets = "all", 
+																							 kegg.exclude = c(), 
+																							 min.size = 1,
```

Comment:
do you think min.size is crucial to include. Maybe we could make it part of the `...` and document it

Reply:
probably not crucial but a nice to have as one of my results only had 3 genes in the set, so I'll see if I can make it part of `...`

Reply:
I don't remember, does the result include a column indicating the gene count for a gene set? If so filtering afterwords could be a much cleaner procedure, as 
- you evaluate afterwards, 
- do not add further arguments, and
-  it would not increase much the running time by definition.

Reply:
No column with gene count in the tibble that I can see but seems to be in the report as NumGenes in format e.g. 67/71. Doing the filtering after would increase the multiple testing a bit but I think it's not essential to include min.size for the moment, we could come back to it later if need be.